### PR TITLE
Rename some targets for consistency

### PR DESCRIPTION
**Pull request by Moohan**
This is a work in progress, and the names may change, but this is a consistent naming convention for now.

### CODE REVIEW

**Comment by Moohan** [comment]
I've split this into 2 commits and have set it to merge to the episode file targets branch. When approved I'll cherry-pick the first commit (which doesn't deal with the episode file) onto `main-R`

File: _targets.R [hunk 1]
```diff
@@ -245,27 +245,27 @@ list(
       format = "rds"
     ),
     ### Target source processed extracts ###
-    tar_target(acute_source_extract, process_extract_acute(
+    tar_target(source_actue_extract, process_extract_acute(
```
Comment (Jennit07) [inline _targets.R:248]:
```suggestion
    tar_target(source_acute_extract, process_extract_acute(
```

File: _targets.R [hunk 2]
```diff
@@ -245,27 +245,27 @@ list(
...
       acute_data,
       year,
       write_to_disk = write_to_disk
     )),
     tar_target(
-      acute_source_extract_tests,
+      source_actue_extract_tests,
```
Comment (Jennit07) [inline _targets.R:254]:
```suggestion
      source_acute_extract_tests,
```

File: _targets.R [hunk 3]
```diff
@@ -245,27 +245,27 @@ list(
...
       process_tests_acute(
-        acute_source_extract,
+        source_actue_extract,
```
Comment (Jennit07) [inline _targets.R:256]:
```suggestion
        source_acute_extract,
```

File: _targets.R [hunk 4]
```diff
@@ -477,21 +477,21 @@ list(
     tar_qs(
       processed_data_list,
       list(
-        source_sc_alarms_tele,
-        acute_source_extract,
-        ae_source_extract,
-        source_sc_care_home,
+        source_actue_extract,
```
Comment (Jennit07) [inline _targets.R:480]:
```suggestion
        source_acute_extract,
```

**Review by Jennit07 — APPROVED** [review]
Looks good, i'll leave you to merge/cherry pick where you were going to!

**Pull request merged**